pom.xml

```diff
@@ -1525,106 +1525,6 @@ under the License.</licenseText>
       <id>reporting</id>
       <reporting>
         <plugins>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-project-info-reports-plugin</artifactId>
```

Comment:
This one should be preserved .... generates common project info

https://maven.apache.org/plugins/maven-project-info-reports-plugin/